### DISCUSSION TITLE
QField Cloud Community Version

### DISCUSSION THREAD

**Discussion by ankitsharma07** [opengisch/QField#3903] [Q&A]
Hi,

I wanted to ask few questions about the Community edition. As far as I have gathered information I can understand that we can have:
**Community edition : Public projects : Multiple Collaborators**
**Pro edition : Private projects : One Contributor**
**Organization edition : Private projects : Multiple Collaborators and paid per Collaborator**

I had one question where in the community edition > public projects as the projects are visible to everyone, can anyone download and  configure the project in QGIS? 

And are the forms visible with the pictures as DCIM folder is available with the public projects.

**Comment by lucienicolier** [comment]
Hi,

About the community project, you can also have private projects.
The limitation will be that you are not able to make private projects collaboration and have 100MB storage limitation.

With public projects, everyone can download the project, but only the owner can configure it.

The pictures are really limited as you have only 100MB storage. 

Best

**Reply by cez2020** [reply]
How can we find and download said public projects?

**Reply by lucienicolier** [reply]
I mean, as downloading is downloading on QField. It is exactly what is a community project for. You can test the existing community projects on your QField and see how it works.

![image](https://user-images.githubusercontent.com/50236352/219588486-80de57d5-b466-49ab-af57-672e93f72e95.png)
